src/sentry/models/integrationfeature.py

```diff
@@ -102,7 +96,7 @@ class IntegrationFeature(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_integrationfeature"
-        unique_together = (("sentry_app", "feature"),)
+        unique_together = (("target_id", "target_type", "feature"),)
```

Comment:
I think you'll need a migration to get this index up

Reply:
Hopefully I can do it in one